docs/manual.md

```diff
@@ -104,27 +115,28 @@ The following docker parameters are used:
   [Running the tool](#running).
 
 
-**Setting an alias.**
-If you are running Apalache on Linux :penguin: or MacOS :green_apple:,
-you can set the handy alias,
-which runs Apalache in docker while sharing the working directory:
+### Setting an alias
+
+If you are running Apalache on Linux :penguin: or MacOS
+:green_apple:, you can define this handy alias in your rc file, which runs
+Apalache in docker while sharing the working directory:
 
 ```bash
 $ alias apalache="docker run --rm -v $(pwd):/var/apalache apalache/mc"
 ```
 
-**Using unstable version of Apalache**
+### Using the unstable version of Apalache
 
 The development of Apalache proceeds at a high pace, and we introduce a
 substantial number of improvements in the unstable branch before the next
-stable release biweekly.  Please refer to [Unstable
-Changes](https://github.com/informalsystems/apalache/blob/unstable/CHANGES.md) and
-[Unstable
-Manual](https://github.com/informalsystems/apalache/blob/unstable/docs/manual.md) for
-the description of new features.  **We recommend using the unstable version if
-you want to try all the exciting new features of Apalache. But be warned: It is
-called "unstable" for a reason**. To use `unstable`, just type
-`apalache/mc:unstable` instead of `apalache/mc` everywhere.
+biweekly stable release. Please refer to the [change
```

Comment:
I would remove "biweekly" completely. I don't think we do have any schedule: sometimes the changes are very fast, sometimes we are distracted by other projects, and there may be no changes for the whole month